app/src/main/res/values/themes.xml

```diff
@@ -3,5 +3,6 @@
     <style name="Theme.TheSocialNetwork" parent="Theme.MaterialComponents.Light.NoActionBar">
         <item name="colorPrimary">@color/colorAccent</item>
         <item name="colorAccent">@color/colorAccent</item>
+        <item name="fontFamily">@font/red_hat_display_regular</item>
```

Comment:
Configure the font in `TheSocialNetwork.TextStyle` as this is our primary text style theme.